### DISCUSSION TITLE
make updating proxy provider easier

### DISCUSSION THREAD

**Discussion by hadifarnoud** [vernesong/OpenClash#3375] [Ideas]
I have included a link to a proxy provider in my personal configuration, but I cannot seem to find a way to update the proxy file manually. The option may be hidden or non-existent. Perhaps changing its position would make it more apparent. Do you think I should open an issue regarding this matter?

**Comment by ghost** [comment]
Do you mean this?
https://dreamacro.github.io/clash/configuration/outbound.html#proxy-providers
So, when you manually update, you should go to the bottom of the proxy page on Yacd or Dashboard.

**Reply by hadifarnoud** [reply]
yes. I'm using remote proxy provider config.

thanks that was what I was looking for
